hack/test/e2e-run.sh

```diff
@@ -34,7 +34,7 @@ run_test_integration_suites() {
 
 run_test_integration_legacy_suites() {
 	(
-		flags="-check.v -check.timeout=${TIMEOUT} -test.timeout=360m $TESTFLAGS"
+		flags="-check.v -check.timeout=${TIMEOUT:=200m} -test.timeout=360m $TESTFLAGS"
```

Comment:
Is it correct that `TIMEOUT` should be assigned here?

Reply:
Not sure, it's "just" the default value :stuck_out_tongue:

Reply:
I think doing `${TIMEOUT:-200m}` will "return" the value of `TIMEOUT` or, if not set, the default value ("200m"), whereas `${TIMEOUT:=200m}` will set the value of `TIMEOUT` to `200m` if `TIMEOUT` wasn't set (see https://unix.stackexchange.com/a/122848/78656)

So if we don't want to change the value of `TIMEOUT` itself, this should probably be `${TIMEOUT:-200m}`